polling_stations/apps/api/postcode.py

```diff
@@ -129,10 +116,9 @@ def retrieve(self, request, postcode=None, format=None, geocoder=geocode, log=Tr
             if ret["polling_station"] and not ret["council"]:
                 ret["council"] = ret["polling_station"].council
 
-        # get custom finder (if no polling station)
+        # this feature doesn't exist any more, but the key is
+        # still in the API response for legacy compatibility
         ret["custom_finder"] = None
```

Comment:
For the sake of not making a breaking API change, I am going to keep the key in the object but it is just always null in 100% of cases.

Reply:
Don't forget that this is basically not a supported API. If the devs.DC code doesn't consume this, then there's nothing to break. If it does, then I suggest we come back and remove this when that code doesn't rely on it.

Reply:
Yeah I think I want to get rid of any code that might be expecting this value to exist in devs.DC first then come back and remove it. In some ways, I may have approach this backwards.